NuRadioReco/modules/trigger/multiHighLowThreshold.py

```diff
@@ -1,21 +1,21 @@
 from NuRadioReco.utilities import units
 from NuRadioReco.framework.parameters import stationParameters as stnp
-from NuRadioReco.framework.trigger import HighLowTrigger
```

Comment:
I'm pretty sure this import is needed. Later a HighLowTrigger object is created.

Reply:
thanks for spotting this! I somehow thought it was an outdated import of an old trigger module... I put it back in